src/nextjs/index.ts

```diff
@@ -108,7 +113,7 @@ export class OttofellerNextjsProject extends NextJsTypeScriptProject {
     new SampleFile(this, 'next-env.d.ts', {sourcePath: path.join(assetsDir, 'next-env.d.ts.sample')})
 
     // ANCHOR ESLint and prettier setup
-    const lintPaths = options.lintPaths ?? ['.projenrc.ts', 'pages', 'src']
+    const lintPaths = options.lintPaths ?? ['.projenrc.ts', 'app', 'src']
```

Comment:
@quesabe, I think we can leave `pages` folder in case there is a incremental adoption applied somewhere

Reply:
I though that any intermediate state  (e.g. during incremental migration) is better be left to a user - the `lintPaths` helps configuring it manually. This way we can leave new projects with a cleaner structure. In addition there is a possibility for `lint` command to fail if `pages` folder does not exist.

Overall I targeted mostly users who spin up new projects. But there are ways to handle both pages router, app router, and something in between - and I planned to add a migration guide to the release notes.